version/version.go

```diff
@@ -0,0 +1,42 @@
+package version
+
+import (
+	"github.com/arduino/arduino-cli/i18n"
+)
```

Comment:
```suggestion
import "fmt"
```
we can remove this and use `fmt` instead